pages/common/gh-browse.md

```diff
@@ -0,0 +1,36 @@
+# gh browse
+
+> Open a GitHub repository in the browser or print the URL.
+> More information: <https://cli.github.com/manual/gh_browse>.
+
+- Open the home page of the current repository in the default web browser:
+
+`gh browse`
+
+- Open the home page of a specific repository in the default web browser:
+
+`gh browse {{owner}}/{{repository}}`
+
+- Open the settings page of the current repository in the default web browser:
+
+`gh browse --settings`
+
+- Open the wiki of the current repository in the default web browser:
+
+`gh browse --wiki`
+
+- Open a specific issue or pull request in the web browser:
+
+`gh browse {{issue_or_pull_request_number}}`
+
+- Open a specific branch in the web browser:
+
+`gh browse --branch {{branch_name}}`
+
+- Open a specific file or directory of the current repository in the web browser (the path should be relative to the repository):
+
+`gh browse {{path/to/file_or_folder}}`
```

Comment:
Do you think we should make a little exception here?
```suggestion
- Open a specific file or directory of the current repository in the web browser:

`gh browse {{path_from_root_of_repository}}`
```

Reply:
I don't like this but I can't see an alternative since anything shorter I think of indicates from / root

Reply:
I could maybe have a `the` between `from` and `root`

Reply:
That's right, @CleanMachine1. This is the root of the *repository*, not the system - hence `{{/path/to/file}}` is not appropriate here.

I think this is the best we can do in this situation.